docs/installing.rst

```diff
@@ -33,7 +33,11 @@ Install the latest development snapshot from the `GitHub repository's master bra
 
 1. Create a new conda environment called ``opencadd``::
 
-    mamba env create -f https://raw.githubusercontent.com/volkamerlab/opencadd/master/devtools/conda-envs/test_env.yaml -n opencadd
+    mamba env create -f https://raw.githubusercontent.com/volkamerlab/opencadd/master/devtools/conda-envs/user_env.yaml -n opencadd-dev
+
+    # Or for contributors:
+    # We recommend to install the test environment, which includes all necessary packages for testing and documentation
+    mamba env create -f https://raw.githubusercontent.com/volkamerlab/opencadd/master/devtools/conda-envs/test_env.yaml -n opencadd-dev
```

Comment:
Here I see a potential source of confusion because we are assigning the same name to two different environments (see line 36). What I suggest is to create a new subsection at the end that you can call "Development version" and then you add this part.
```suggestion
To install the development version of OpenCADD, you can run::
    mamba env create -f https://raw.githubusercontent.com/volkamerlab/opencadd/master/devtools/conda-envs/test_env.yaml -n opencadd-dev
```
And then instructions to activate and how to run tests if needed, etc. IMO, showing a separate subsection for dev version is very helpful for the reader.

Reply:
Wonderful, I like this a lot! Please check if this is what you had in mind:
https://opencadd.readthedocs.io/en/joss-review-issue-127/installing.html#development-version

Reply:
Looks very cool. Thanks for adding this!